### DISCUSSION TITLE
How to get list of consumers with tags for a specific queue

### DISCUSSION THREAD

**Discussion by ikarol** [php-amqplib/php-amqplib#911] [Q&A] [answered]
Hi, I'm looking for something like RabbitMQ cli tool has - https://www.rabbitmq.com/rabbitmqctl.8.html#list_consumers

This is needed in order to get the list of consumers with their tags.
The end goal is cancellation of specific consumers via basic_cancel

If it's not currently possible, could it be integrated later into the library as a new feature?

Any advices would be appreciated

**Comment by ramunasd** [comment, marked as answer]
This library implements AMQP 0.9 protocol which is publisher/consumer oriented and does not provide such info.

Very likely You won't that kind of information without management plugin. Please take a look at API - You can get consumer list from it.

https://pulse.mozilla.org/api/
